src/renderer/components/App/AppSidemenu/AppSidemenuSettings.vue

```diff
@@ -344,7 +367,7 @@ export default {
     },
 
     emitClose () {
-      if (this.outsideClick && !(this.isResetDataModalOpen || this.isScreenshotProtectionModalOpen)) {
+      if (this.outsideClick && !(this.isResetDataModalOpen || this.isScreenshotProtectionModalOpen || this.isManageBlacklistModalOpen)) {
```

Comment:
I would consider moving this to a computed property due to the line length

Reply:
Fixed!